nixos/modules/config/fonts/fontdir.nix

```diff
@@ -5,14 +5,16 @@ with lib;
 let
 
   x11Fonts = pkgs.runCommand "X11-fonts" { preferLocalBuild = true; } ''
+    font_formats=('ttf' 'otf' 'pcf' 'pfa' 'pfb' 'pcf' 'bdf')
```

Comment:
You wrote `pcf` two times.

Reply:
ups, sorry about that, I amended the commit